docs/index.md

```diff
@@ -1,3 +1,3 @@
 # mongodb-vapor Documentation Index
-- [1.0.0-beta.0 (current)](current/index.html)
-- [1.0.0](1.0.0/index.html)
+- [1.0.0-beta.0](1.0.0-beta.0/index.html)
```

Comment:
maybe we ought to just filter the beta release out of the documentation altogether, since it might be a bit confusing if it appears in the list before the 1.0.0 version

Reply:
that seems reasonable to me. maybe rather than editing the script I should just delete that folder manually and re-generate the docs?
there could be a time in the future where we are doing another release and do a beta first, and want the beta docs to be accessible before tagging GA

Reply:
Yeah that SGTM

Reply:
deleted the beta docs, put the script back in original state, and regenerated the index